security/authzero/AuthZeroClient.ts

```diff
@@ -88,14 +101,16 @@ export class AuthZeroClient extends BaseOAuthClient<AuthZeroClientConfig, AuthZe
         const appState = this.captureRedirectState();
         await this.client.loginWithRedirect({
             appState,
-            authorizationParams: {scope: this.loginScope}
+            authorizationParams: this.buildAuthParams({scope: this.loginScope})
         });
         await never();
     }
 
     protected override async doLoginPopupAsync(): Promise<void> {
         try {
-            await this.client.loginWithPopup({authorizationParams: {scope: this.loginScope}});
+            await this.client.loginWithPopup({
+                authorizationParams: this.buildAuthParams({scope: this.loginScope})
```

Comment:
Is it not the case that it will get the default audience as now specified in `createClient`?  Just as it will get the redirect_url?    If so, I think the redundant calls to buildAuthParams is confusing.

Reply:
Fixed thanks